<!DOCTYPE html>
<html>
<head>
  <title>Amit Chaudhary - Profile</title>
  <style>
    body {
      font-family: Arial, sans-serif;
      margin: 20px;
      background-color: #f1f1f1;
    }

    h1 {
      color: #333;
      text-align: center;
      font-size: 36px;
      margin-top: 40px;
    }

    p {
      margin-bottom: 10px;
    }

    .section {
      margin-bottom: 30px;
      background-color: #fff;
      padding: 20px;
      border-radius: 5px;
      box-shadow: 0 2px 5px rgba(0, 0, 0, 0.1);
    }

    .section-title {
      font-weight: bold;
      margin-bottom: 10px;
      color: #333;
      font-size: 24px;
    }
    
    .profile-photo {
      display: block;
      margin: 0 auto;
      border-radius: 50%;
      box-shadow: 0 2px 5px rgba(0, 0, 0, 0.3);
      margin-bottom: 20px;
    }
  </style>
</head>
<body>
  <h1 style="color: #007bff;">Amit Chaudhary, Ph.D.</h1>
  
  <img src="C:\Users\Bioinfo\Desktop\photo.jpg" alt="Profile Photo" width="200" height="200" class="profile-photo">
  
  <div class="section">
    <div class="section-title">Career Objective</div>
    <p>Seeking a challenging position in the field of genomics, proteomics, structural biology, and network biology, where I can utilize my expertise to contribute to scientific advancements and make a positive impact on human health. Committed to expanding knowledge and employing innovative approaches to unravel complex biological systems and discover novel therapeutic targets. Dedicated to continuous professional growth and collaboration within multidisciplinary teams to drive cutting-edge research and contribute to the advancement of the life sciences field.</p>
  </div>
  
  <div class="section">
    <div class="section-title">Education and Training</div>
    <ul>
      <li>
        <strong>Doctor of Philosophy (Biotechnology)</strong><br>
        Motilal Nehru National Institute of Technology, Allahabad (India)<br>
        Date of the thesis defense: 16-12-2020<br>
        Date of Award: 30-12-2020<br>
        Title of the thesis: Studies on genetic factors involved in Alzheimer’s disease using an integrated computational approach
      </li>
      <li>
        <strong>Master in Technology (Bioinformatics)</strong><br>
        Indian Institute of Information Technology, Allahabad, Allahabad (India)
      </li>
      <li>
        <strong>Bachelor of Pharmacy</strong><br>
        United Institute of Pharmacy, Allahabad (India)
      </li>
    </ul>
  </div>

  <div class="section">
    <div class="section-title">Experience</div>

  <ul>
    <li>
      <strong>2020-2021, June</strong> - Scientist, Amity Institute of Biotechnology, Amity University, Noida<br>
      <strong>Project title:</strong> mRNA Vaccine Project<br>
      <strong>Funding agency:</strong> Kleberg Foundation and Baylor College of Medicine, Houston, TX
    </li>
    <li>
      <p><strong>2021-Present</strong> - Institute Postdoctoral Fellow, Indian Institute of Technology Bombay (IIT Bombay)<br></p>
      <strong>Projects:</strong>
      <ol>
        <li>Investigating Antibacterial and Amyloidgenicity properties of Antimicrobial peptide (A Molecular dynamics simulation approach).</li>
        <li>Exploring Alpha-synuclein secondary structure transition (A Molecular dynamics simulation approach).</li>
        <li>Glucose Biosensor fabrication.</li>
      </ol>
    </li>
  </ul>
  
</div>


  <div class="section">
    <div class="section-title">Publications</div>
    <p><strong>SCI Journals/Web of Science/Scopus articles</strong></p>
    <ol>
      <li>Chaudhary, A., Yadav, B. S., Singh, S., Maurya, P. K., Mishra, A., Srivastva, S., & Mani, A. (2017). Docking-based screening of Ficus religiosa phytochemicals as inhibitors of human histamine H2 receptor. Pharmacognosy magazine, 13(Suppl 3), S706. (IF-1.085)</li>
      <li>Chaudhary, A., & Mani, A. (2020). Screening of potential inhibitors against flotillin-1 as therapeutics for Alzheimer's disease. Pharmacognosy Magazine, 16(69), 274-281. (IF-1.085)</li>
      <li>Pahal, S., Chaudhary, A., & Singh, S. (2022). Screening of Natural Compounds Against SOD1 as a Therapeutic Target for Amyotrophic Lateral Sclerosis. Letters in Drug Design & Discovery, 19(10), 877-887. (IF- 1.15)</li>
      <li>Chaudhary, A., & Mani, A. (2022). Aβ peptides stabilize GPCRs in inactive form and trigger inverse agonism in Alzheimer's disease. Biochimie, 201, 75-78. (IF-4.372)</li>
      <li>Chaudhary A, Chaurasia PK, Kushwaha S, Chauhan P, Chawade A, Mani A. Correlating multi-functional role of cold shock domain proteins with intrinsically disordered regions. Int J Biol Macromol. 2022 Nov 1;220:743-753. doi: 10.1016/j.ijbiomac.2022.08.100. Epub 2022 Aug 18. PMID: 35987358. (IF- 8.025)</li>
      <li>Chaudhary, A., Shikha Kushwah, Neha Shree Maurya, and Ashutosh Mani. "Insights from RNA-Seq analysis of Alzheimer's data suggest upregulation of GPCRs." Gene Reports 21 (2020): 100921. (IF-  1.51)</li>
      <li>Chaudhary A, Singh V, Varadwaj PK, Mani A. Screening natural inhibitors against upregulated G-protein coupled receptors as potential therapeutics of Alzheimer's disease. J Biomol Struct Dyn. 2022 Feb;40(2):673-684. doi: 10.1080/07391102.2020.1817784. Epub 2020 Sep 9. PMID: 32900274. (IF-4.15)</li>
      <li>Kumari, R., Chaudhary, A., & Mani, A. (2020). Casuarictin: A new herbal drug molecule for Alzheimer's disease as inhibitor of presenilin stabilization factor like protein. Heliyon, 6(11), e05546. (IF- 2.85)</li>
      <li>Pahal, S., Gupta, A., Choudhary, P., Chaudhary, A., & Singh, S. (2021). Network pharmacological evaluation of Withania somnifera bioactive phytochemicals for identifying novel potential inhibitors against neurodegenerative disorder. Journal of Biomolecular Structure and Dynamics, 40(21) 1-12. (IF: 4.152)</li>
      <li>Maurya, P. K., Mishra, A., Yadav, B. S., Singh, S., Kumar, P., Chaudhary, A., & Mani, A. (2017). Role of Y Box Protein-1 in cancer: As potential biomarker and novel therapeutic target. Journal of Cancer, 8(10), 1900. (IF- 4.478)</li>
      <li>Rawal, K., Sinha, R., Abbasi, B. A., Chaudhary, A., Nath, S. K., Kumari, P., ... & Bottazzi, M. E. (2021). Identification of vaccine targets in pathogens and design of a vaccine using computational approaches. Scientific reports, 11(1), 1-25. (IF-4.996)</li>
      <li>Murugesan, S. N., Yadav, B. S., Maurya, P. K., Chaudhary, A., Singh, S., & Mani, A. (2019). Interaction network analysis of YBX1 for identification of therapeutic targets in adenocarcinomas. Journal of Biosciences. Ref.:  Ms. No. JBSC-D-18-00271R1. (IF-2.795)</li>
      <li>Nath, S. K., Pankajakshan, P., Sharma, T., Kumari, P., Shinde, S., Garg, N., ... & Rawal, K. (2023). A Data-Driven Approach to Construct a Molecular Map of Trypanosoma cruzi to Identify Drugs and Vaccine Targets. Vaccines, 11(2), 267. (See Acknowledgement section)</li>
    </ol>
    <p><strong>Non-SCI Journals</strong></p>
    <ol start="14">
      <li>Chaudhary, A., Maurya, P. K., Yadav, B. S., Singh, S., & Mani, A. (2018). Current Therapeutic Targets for Alzheimer’s Disease. Journal of Biomedicine, 3, 74-84. (IF- NA)</li>
      <li>Chaudhary, A., Tripathi, R., Jain. Y., &Guttula, S., "Antioxidant therapies to oxidative stress related neurotrophins involved in Azheimer’s disease". European Journal of Pharmaceutical and Medical Research, April 2015, volume 2(2015):245-261. (SJIF IF- 6.2)</li>
      <li>Singh, Y., Patel, H., Chaudhary, A., & Deepa, S. (2021). Utilizing Zebrafish Animal Model as a Helping Hand for COVID-19 Infection. Coronaviruses, 2(2), 272-274. (IF- NA)</li>
    </ol>
    <p><strong>Conference preceding</strong></p>
    <ol start="17">
      <li>Srivastava, S., Chaudhary, A., & Mani, A. (2018, October). Identification of potential inhibitors for identified novel drug targets of Aspergillus fumigatus. In 2018 International Conference on Bioinformatics and Systems Biology (BSB) (pp. 81-84). IEEE.</li>
    </ol>

    <p><strong>Non- SCI Journals</strong></p>
    <ol start="18">
    <li>Chaudhary, A., Maurya, P. K., Singh, S., & Mani, A. (2023). Targeting of Ficus religiosa phytochemicals against cancer: in silico docking, molecular dynamics, and ADMET approaches. Journal of Ayurveda and Integrative Medicine, In Press. (IF-2.196)</li>
    <li>Srivastava, S., Chaudhary, A., & Mani, A. (2018, October). Identification of potential inhibitors for identified novel drug targets of Aspergillus fumigatus. In 2018 International Conference on Bioinformatics and Systems Biology (BSB) (pp. 81-84). IEEE. (Conference Proceeding)</li>
    </ol>

<strong><p>Book Chapters</strong></p>
    <ol start="20">
    <li>Maurya, P.K., Chaudhary, A., Singh, S., Mani, A. Signal Networks: Information flow, Computation and Decision Making. Concepts in Cell Signaling ISBN 978-93-84502-47-8 (PPH New Delhi) (2017)</li>
    <li>Singh Y., Patel H., Deepa., Chaudhary A., Salahddine K., Hu W. "The challenge ahead on integrating the path of machine learning into the field of cardiac imaging." Internet of Everything and Big data: Major challenges in smart cities, Taylor & Francis.</li>
    <li>Maurya N., Chaudhary, A., Mani, P., Mani A., and Srivastava, S. K. Chapter 12. Artificial Intelligence: Prospects in Drug Discovery and Health Technology. In Recent Advances in Computer Aided Drug Designing, Ashutosh Mani & Akhil Varshney(Eds) ISBN: 978-1-53619-739-6. Nova Science Publishers Inc., New York (2021)</li>
    <li>Chaudhary, A., Varshney, A., Mani A., and Srivastava, S. K. Chapter 2. Computational Approaches for Drug Screening and Pharmacokinetic Studies. In Recent Advances in Computer Aided Drug Designing, ISBN: 978-1-53619-739-6. Nova Science Publishers Inc., New York (2021)</li>
    </ol>

<strong><p>Communicated Articles</strong></p>
    <ol start="25">
    <li>Pahal, S., Chaudhary, A., & Mani, A. "Designing a multivalent epitope-based vaccine against Cunninghamella bertholletiae: An in silico approach" (Communicated).</li>
    <li>Singh P., Kaushik M., Gupta S., Chaudhary, A. "Unlocking the Mysteries of Alzheimer's Disease: Role of Single-Cell Sequencing in Uncovering Microbial Infections" (Communicated).</li>
  </ol>

<strong><p>Articles Under Preparation</strong></p>
    <ol start="27">
    <li>Chaudhary, A., Prasad, A., Panwar, A. S. "Investigating Antibacterial and Amyloidgenicity properties of Antimicrobial peptide: A Molecular dynamics simulation approach." (2023)</li>
    <li>Maity, A., Chaudhary, A., Panwar, A. S., Sharma, K. "Exploring Alpha synuclein secondary structure transition in presence of polymers." (2023)</li>
    <li>Makare, A., Chaudhary, A., Deshpande, P., Banerjee D. "Screening of Glucose binding proteins for Glucose Biosensor fabrication." (2023)</li>
  </ol>

<strong><p>Preprints</strong></p>
    <ol start="30">
    <li>Rawal, K., Chaudhary, A., Shinde, S., Jain, A., et al. (2023) "A general map of Chagas disease and their subnetworks." (Preprint)</li>
  </ol>
 </div>

  <div class="section">
    <div class="section-title">Area of Interest</div>
  <ul>
    <li>Genomics and Proteomics</li>
    <li>RNA-Seq and Chip-Seq data analysis</li>
    <li>DNA methylation</li>
    <li>Molecular dynamics simulation (cMD, aMD, SMD, Umbrella sampling)</li>
    <li>Molecular docking of macromolecules (Protein-ligand, Protein-Protein, Protein-Nucleic acid)</li>
    <li>Biological database (DDBJ, NCBI, EBI, Uniprot, EMBL)</li>
    <li>Secondary structure prediction of protein</li>
    <li>Tertiary structure prediction (Ab initio, Homology modeling of protein)</li>
    <li>Pharmacokinetics parameters studies (ADMET) of molecules</li>
    <li>Network biology and pathway enrichment analysis of genes and proteins</li>
    <li>Phylogenetic analysis of proteins and nucleic acids</li>
    <li>Text mining and data curation</li>
  </ul>

 </div>

  <div class="section">
    <div class="section-title">Bioinformatics Tools</div>
  <ul>
    <li>Indexing and quantification tools (Salmon)</li>
    <li>Mapping Tools (Bowtie, BWA)</li>
    <li>R/Bioconductor Packages (DeSeq2, edgeR, Ballgown)</li>
    <li>Fragpipe</li>
    <li>Maxquant</li>
    <li>SMARTseq</li>
    <li>Bismark</li>
    <li>Network Analysis Tools (Cytoscape, Gephi)</li>
    <li>MODELLER</li>
    <li>Binding Site Identification Tools</li>
    <li>Docking & Molecular Dynamics Simulation (Schrodinger suites: GLIDE, AutoDock4, AutoDOCK Vina, Desmond, GROMACS, NAMD)</li>
    <li>Cheminformatics tools (Discovery Studio, Chemsketch)</li>
    <li>Phylogenetic analysis tools (MEGA, Phylo.io, Figtree, BEAST)</li>
    <li>Cell Designer</li>
    <li>VMD</li>
    <li>CHARMMGUI</li>
    <li>PACKMOL</li>
  </ul>
 </div>


<div class="section">
      <div class="section-title">Projects</div>
  <style>
    .justified {
      text-align: justify;
      margin-bottom: 0; /* Remove bottom margin */
      padding-bottom: 0; /* Remove bottom padding */
    }
  </style>
  
  <h3>Bachelor's project</h3>
  <p class="justified"><strong>Project title:</strong> A Project Report on Standardization of herbal drugs (Trikatu)</p>
  <p class="justified"><strong>Project supervisor:</strong> Mrs. Geeta Rai</p>
  <p class="justified"><strong>Summary:</strong> As part of my undergraduate final semester thesis, I undertook a project focused on standardizing an herbal drug known as Trikatu. Specifically, I worked on standardizing and validating the Ayurvedic formulation called "Trikatu Churna." To achieve this objective, we conducted various wet lab experiments to assess the quality and purity of the herbal drug. Additionally, we also investigated the nature of any adulterants present, utilizing parameters such as morphological, microscopic, physical, chemical, and biological analysis. Our study revealed that the herbal drug formulation adhered to the standards outlined in the Ayurvedic Pharmacopoeia of India. As a result, we concluded that these formulations are therapeutically safe and can be effectively utilized in the treatment of various ailments. Overall, our project contributes to the broader effort of enhancing the reliability and efficacy of Ayurvedic medicines through standardized processes and validation procedures.</p>
  

  <h3>Master's project</h3>
  <p class="justified"><strong>Thesis Title:</strong> Antioxidant Therapies to Oxidative stress-related Neurotrophins Involved in Alzheimer’s Disease</p>
  <p class="justified"><strong>Supervisor:</strong> Dr. Satyavani Guttula</p>
  <p class="justified"><strong>Summary:</strong> My Master's research is focused on the role of antioxidants and their defense mechanisms in oxidative stress implicated in Alzheimer's disease. The study utilized in silico methods to identify the binding affinity of antioxidants against neurotrophins, and ADMET analysis was performed to examine the pharmacokinetic parameters of these antioxidants. The docking results of our study revealed that Theaflavin, Catechin, Luteolin, Kaemferol, Gallic acid, Myricetin, Eriodictyol, Uric acid, Ascorbic and Isorhamnetin acid showed the best binding affinity with three different proteins (1NT3, 1BND, 1B8M). This finding suggests that these compounds could effectively reduce oxidative stress and play an important role in reducing the risk or delaying the clinical onset of progressive dementia.</p>
  

  <h3>Ph.D. Project</h3>
  <p class="justified"><strong>Title:</strong> Studies on genetic factors involved in Alzheimer’s disease using an integrated computational approach</p>
  <p class="justified"><strong>Supervisor:</strong> Dr. Ashutosh Mani</p>
  <p class="justified"><strong>Summary:</strong> Dementia is increasing rapidly in the world population. Alzheimer's disease is an important factor behind the development of dementia in older people. Due to the unclearly known mechanism of pathophysiology and target identification Alzheimer's disease treatment remains a great challenge for modern drug discovery. Only a few drugs are available for the treatment of the disease. There is an urgent need to explore these targets for designing efficient Alzheimer's drugs with minimum side effects. In this study, RNA Seq analysis was performed to identify the differentially expressed genes, and their significant enriched functions with pathways were identified by using an integrated computational approach. On the basis of the literature review survey, a few upregulated genes were selected for further studies. Template-based modelling was performed to generate the 3D structure of upregulated receptors. Molecular docking and molecular dynamics simulation studies of docked complexes followed by pharmacokinetic parameters analysis of compounds helped in identifying promising lead compounds. Analysis of RNA-Seq data in this study was an important source for providing information regarding the significant biomarkers, diagnostic as well as novel therapeutic targets for AD. We conclude that expression of major neurotransmitters is increased due to inverse agonism, however these neurotransmitters are not available to bind with their natural ligands and are preoccupied with misfolded Aβ granules. This condition further disturbs calcium signaling and promotes NFT formation and finally causes neuronal cell death. These results further provide a basis for evaluating and exploring the functions/pathways of DEGs, biomarker development, and identification of novel drug targets in Alzheimer's disease. In the third objective, inhibitors for upregulated genes' proteins were designed. Natural inhibitors of these receptors are prospective therapeutics for Alzheimer's disease because they will not only compete with Aβ granules for binding with GPCRs but also work as potential breakers of the negative agonism loop. Overall predicted targets and screened compounds could be helpful for the AD drug discovery process and the treatment of AD patients. The current in silico study could be suitable for further designing novel inhibitors against target proteins with the help of in vivo and in vitro analysis.</p>
 </div>


<div class="section">

  <div class="section-title">Courses</div>
  <ul>
    <li>Pharmaceutical Science</li>
    <li>Fundamentals of Computers</li>
    <li>Data Structure and Algorithms</li>
    <li>Introduction to Genomics and Proteomics</li>
    <li>Mathematics and Biostatistics</li>
    <li>Language Algorithm and Tools</li>
    <li>Computer Aided Drug Design</li>
    <li>Molecular Structure Prediction and Visualization</li>
    <li>Systems Biology</li>
    <li>Chemoinformatics</li>
    <li>Advanced Bioinformatics</li>
    <li>Medical Biotechnology</li>
    <li>Metabolic Engineering</li>
    <li>Machine Learning</li>
  </ul>
   </div>

<div class="section">
<div class="section-title">Conferences</div>
  <ul>
    <li>Presented Poster presentation in International Conference on Translational Biotechnology (BioSangam 2016)</li>
    <li>Participated in International Conference on Bioinformatics and Systems Biology & Workshop on Systems Biology 26th-28th October 2018 at IIIT Allahabad</li>
  </ul>
  
  <div class="section-title">Seminars</div>
  <ul>
    <li>Participated in “One-day Symposium on Network Biology” held at Center for Computational Biology IIIT-Delhi on 28th Oct 2017</li>
    <li>Delivered poster presentation in “Symposium on Accelerating Biology 2018: Digitizing Life”, Indian Institute of Science Education and Research (IISER), Pune, Maharashtra (January 9-11, 2018)</li>
    <li>Participated in the “Bioinformatics Workshop on Proteomics & Drug Design” at the Supercomputing Facility for Bioinformatics & Computational Biology, Indian Institute of Technology, Delhi</li>
    <li>Participated in 10 days GIAN course on "Genomics, Personalized Medicine and Ethics" held at MNNIT on July 31, 2017 – August 11, 2017</li>
  </ul>
</div>
  <div class="section">
    <div class="section-title"> Database</div>


  <strong><p>Glucose Binding Proteins Database (Unpublished)</strong></p>
  
  <h3>About</h3>
  <p>The Glucose Binding Proteins Database is a comprehensive collection of information related to proteins that bind to glucose molecules. It aims to provide researchers and scientists with a valuable resource for studying and understanding the various aspects of glucose binding proteins.</p>
  
  <h3>Features</h3>
  <ul>
    <li>Curated data on glucose binding proteins</li>
    <li>Structural information and binding site details</li>
    <li>Search and browse capabilities</li>
  </ul>
  
  <h3>Accessing the Database</h3>
  <p>Please note that the Glucose Binding Proteins Database is currently unpublished and not publicly accessible. It is under development and undergoing continuous refinement. Researchers and interested parties may contact the database administrators for more information or potential collaborations.</p>
  
  <p><h3>Contact Information</h3>
  For inquiries and collaborations, please contact:
  <strong><p>Name:</strong></p> Dr. Amit Chaudhary
  <strong><p>Email:</strong></p> amitunited0532@gmail.com</p>
</div>

</body>
</html>
